service/topology_coordinator.cc

```diff
@@ -2481,6 +2481,12 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                     node = retake_node(co_await start_operation(), node.id);
                 }
 
+                // Make decomissioning node a non voter before reportiing operation completion below.
+                // Otherwise the decommissioned node my see the completion and exit before it is removed from
+                // the config at which point the removal from the config will hang if the claster had only two
```

Comment:
just some small typos:

```suggestion
                // Make decommissioning node a non voter before reporting operation completion below.
                // Otherwise the decommissioned node may see the completion and exit before it is removed from
                // the config at which point the removal from the config will hang if the cluster had only two
```